### ISSUE TITLE
History size constraints

### ISSUE THREAD

**Issue by amivaleo** [Tudmotu/gnome-shell-extension-clipboard-indicator#102]
Hi!
Can you just increase the constraint for the history size, from 50 to, let's say, 100?
Sometimes 50 is just not enough for me, and I had to change this constraint locally in your scripts.

Thanks. :)

**Comment by sT331h0rs3** [comment]
I agree, I used to set history size to 100-150. It would be really nice to have such option.

**Comment by amivaleo** [comment]
I can't set the history size to any value higher than 50 yet. :(
